src/actions/resize/crop.ts

```diff
@@ -1,17 +1,29 @@
 import '../../interfaces/IAction';
 import Action from "../Action";
+import {IAction} from "../../interfaces/IAction";
 
 /**
  * @implements IAction
  */
 class Crop extends Action implements IAction {
```

Comment:
This is only initial, right? 
cause lots of things are missing - gravity, ar, etc.

Reply:
Yes, we'll be working through the full compilation tests anyway, this is a proof of concept.